manifests/testing-framework/testing-framework.yaml

```diff
@@ -30,6 +30,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: AUTO_CLEAN_UP
+              valueFrom:
+                configMapKeyRef:
+                  name: env
+                  key: AUTO_CLEAN_UP
```

Comment:
this key needs to be marked as optional, otherwise the pods might fail to start in case it's missing (manual triggers).

refs:
* https://github.com/kubernetes/website/pull/35160/files#diff-20d118b7f6f20eeff74747a63494a459bf067d00cb3fe543b858ec8222b754c8R702
* https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#optional-configmaps